app/uk/gov/hmrc/economiccrimelevyaccount/views/PaymentsView.scala.html

```diff
@@ -70,7 +70,7 @@
                 HeadCell(content = Text(messages("account.viewPayments.table.paymentsHistory.header.3")), classes = "govuk-table__header govuk-!-width-one-quarter"),
                 HeadCell(content = Text(messages("account.viewPayments.table.paymentsHistory.header.4")), classes = "govuk-table__header govuk-!-width-one-quarter"),
                 HeadCell(content = Text(messages("account.viewPayments.table.paymentsHistory.header.5")), classes = "govuk-table__header govuk-!-width-one-quarter"),
-                HeadCell(content = Text(messages("account.viewPayments.table.paymentsHistory.header.6")), classes = "govuk-table__header govuk-!-width-one-quarter")
+                HeadCell(content = if (disableRefund) " " else Text(messages("account.viewPayments.table.paymentsHistory.header.6")), classes = "govuk-table__header govuk-!-width-one-quarter")
```

Comment:
Can this not match \&nbsp; as you did below?

Reply:
I tried it but you get the literal "&nbsp;" so I had to use a space.